src/lambdapi.ml

```diff
@@ -333,6 +355,11 @@ let check_cmd =
   Term.(const check_cmd $ global_config $ timeout $ recompile $ files),
   Term.info "check" ~doc ~man:man_pkg_file
 
+let decision_tree_cmd =
+  let doc = "Prints decision tree of a symbol to standard output." in
```

Comment:
Maybe give a bit more details about what format? Also give some help for visualising the result.